src/test/java/com/icthh/xm/ms/dashboard/web/rest/AtomicBulkDashboardResourceIntTest.java

```diff
@@ -64,18 +65,25 @@ public void setup() {
 
     @Test
     @SneakyThrows
+    @Transactional
     public void shouldAtomicCreateDashboards() {
         httpMock.perform(post("/api/dashboards/bulk")
             .contentType(APPLICATION_JSON)
             .content(readAsString("bulkAtomicCreateDashboards.json")))
             .andExpect(status().isOk());
 
         assertTrue(dashboardRepository.findAll().stream()
-            .anyMatch(dashboard -> "Bulk atomic crate first".equalsIgnoreCase(dashboard.getName()))
+            .anyMatch(dashboard ->
+                "Bulk atomic crate first".equalsIgnoreCase(dashboard.getName())
+                    && !dashboard.getWidgets().isEmpty()
```

Comment:
asserts must be simple, this assert too complex to find fail reason

Reply:
updated

Reply:
@sergeysenja1992